addons/mail/static/src/chatter/web/mail_composer_send_dropdown.js

```diff
@@ -22,7 +22,9 @@ class MailComposerSendDropdown extends Component {
         this.orm = useService("orm");
     }
 
-    async onClickSend() {
+    async onClickSend(ev) {
+        const sendButton = ev.target;
+        sendButton.setAttribute("disabled", true);
```

Comment:
You could also use a state to handle this and add in the template a dynamic attribute on the button based on it.
I am personally not a big fan of changing attributes on the fly in the methods, unless absolutely necessary. 

What happens if the target is no longer in the DOM?
Your change wouldn't be taken into account or you could even have crashes.